man/man8/funcslower.8

```diff
@@ -2,7 +2,7 @@
 .SH NAME
 fileslower \- Trace slow kernel or user function calls.
```

Comment:
since you are touching this file, could you change here `fileslower` to `funcslower`?